core/players-handbook-2024/archetypes/monk-open-hand.xml

```diff
@@ -0,0 +1,125 @@
+		<description>
+			<p class="flavor">Master Unarmed Combat Techniques</p>
+			<p>Warriors of the Open Hand are masters of unarmed combat. They learn techniques to push and trip their opponents and manipulate their own energy to protect themselves from harm.</p>
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE"/>
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_WHOLENESS_OF_BODY"/>
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_FLEET_STEP"/>
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_QUIVERING_PALM"/>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE" level="3"/>
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_WHOLENESS_OF_BODY" level="6"/>
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_FLEET_STEP" level="11"/>
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_QUIVERING_PALM" level="17"/>
+		</rules>
+	</element>
+
+	<element name="Level 3: Open Hand Technique" type="Archetype Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE</requirements>
+		<description>
+			<p>Whenever you hit a creature with an attack granted by your Flurry of Blows, you can impose one of the following effects on that target.</p>
+			<p class="indent"><em><strong>Addle.</strong></em> The target can't make Opportunity Attacks until the start of its next turn.</p>
+			<p class="indent"><em><strong>Push.</strong></em> The target must succeed on a Strength saving throw or be pushed up to 15 feet away from you.</p>
+			<p class="indent"><em><strong>Topple.</strong></em> The target must succeed on a Dexterity saving throw or have the Prone condition.</p>
+		</description>
+		<sheet action="Flurry of Blows" alt="Open Hand Technique">
+			<description>Whenever you hit a creature with an attack granted by your Flurry of Blows, you can impose one of the following effects on that target.</description>
+		</sheet>
+		<rules>
+			<select type="Archetype Feature" name="Addle" supports="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_ADDLE" default="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_ADDLE"/>
+			<select type="Archetype Feature" name="Push" supports="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_PUSH" default="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_PUSH"/>
+			<select type="Archetype Feature" name="Topple" supports="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_TOPPLE" default="ID_WOTC_PHB24_ARCHETYPE_FEATURE_MONK_OPEN_HAND_OPEN_HAND_TECHNIQUE_TOPPLE"/>
+		</rules>
```

Comment:
```suggestion
		<sheet alt="Open Hand Technique">
			<description>On hit using your Flurry of Blows, you can force one of the following effects on the target: &#13;
			• It can't make Opportunity Attacks until the start of its next turn. &#13;
			• It must succeed on a DC{{monks focus:dc}} Strength saving throw or be pushed up to 15 ft. away from you. &#13;
			• It must succeed on a DC{{monks focus:dc}} Dexterity saving throw or have the Prone condition.</description>
		</sheet>
```